nemo/collections/asr/modules/rnnt.py

```diff
@@ -384,6 +384,13 @@ def batch_replace_states_mask(
         # same as `dst_states[0][mask] = src_states[0][mask]`, but non-blocking
         torch.where(mask.unsqueeze(-1), src_states[0], dst_states[0], out=dst_states[0])
 
+    @classmethod
```

Comment:
`AbstractRNNTDecoder` already has `batch_replace_states_all` method. 
`RNNTDecoder` and `StatelessTransducerDecoder` have different types of states: `tuple` for RNN-T and `list` for Stateless (but there is confusion in the code since, in most cases, the `list` type works, but not all).
I prefer to keep it clean and provide implementation for both classes. Also, due to planned work on the Transformer-Decoder, there will be a separate state type for the Transformer.

Reply:
Makes sense